BUILD.md

```diff
@@ -425,7 +410,7 @@ builds.
 ### Linux Development Environment Requirements
 
 This repository has been built and tested on the two most recent Ubuntu LTS
-versions. Currently, the oldest supported version is Ubuntu 16.04, meaning
+versions. Currently, the oldest supported version is Ubuntu 18.04, meaning
```

Comment:
Thought we only supported current LTS releases (ie: 20/22)?

Reply:
Oh hmmm well yes, that is what we should be supporting, but right now the github actions runners are 18 and 20. And I don't think internal CI has 22 set up just yet. Though I could be running on outdated info.